scripts/prepare_disk.sh

```diff
@@ -128,6 +176,16 @@ if [ ! -r "${PARTITION_TABLE_FILE}" ]; then
     exit 1
 fi
 
+if [ ! -b "${TARGET_DISK}" ]; then
+    echo "Error, block device '${TARGET_DISK}' does not exist."
+    exit 1
+fi
```

Comment:
nice catch, i'm pretty sure at some point this was there, i must have miss-merged it at some point ...